frontend/src/stories/PanelStack/PanelStack.css

```diff
@@ -0,0 +1,14 @@
+.bp3-panel-stack {
+	height: 200px;
+}
+
+.panel-stack-content {
+	padding: 8px;
+	display: flex;
+    align-items: center;
```

Comment:
I don't know if I'm overly pedantic or this is a super-subtle joke, but align... isn't aligned :)

Reply:
Oh, that's completely accidental XD